scripts/script.js

```diff
@@ -0,0 +1,53 @@
+ 
+function cost() {
+    alert("$123.45");
+}
+
+function infoMurphy(){
+    alert("Murphy \nBreed: Boxer \n\nAdoption fees: $123.45");
+}

```

Comment:
Functions are designed to be reused. In this case, you should only have one info function that works for all dogs. You'd do this by creating a function with info: `func total(cost, dog, breed)`

Reply:
Great!! now, that makes sense, less and more effective code. As you suggested I change the name of the function and add some values, parameters and arguments, to wrap the function. Forgot to mention the sum of the totals when clicking the 'adopt' button. Thanks for your help.